src/sensors-app.html

```diff
@@ -206,13 +206,13 @@
                     var i = 0;
                     var needsRelayout = typeof this.readingTimestamp == "undefined";
 
-                    for (property in this.reading) {
-                        if (property == 'timeStamp') {
-                            this.readingTimestamp = 'timeStamp: ' + round(this.reading.timeStamp, 3);
+                    for (property in this) {
+                        if (property == 'timestamp') {
+                            this.readingTimestamp = 'timestamp: ' + round(this.timestamp, 3);
                             this.app.notifyPath('sensorDataModel.' + this.id + '.readingTimestamp');
                         } else {
                             var readingId = 'readingValue' + i++;
-                            this[readingId] = property + ': ' + round(this.reading[property], 3);
+                            this[readingId] = property + ': ' + round(this[property], 3);
```

Comment:
This would not work anymore. You can check at https://v-two-dot-genericsensors.appspot.com/